releases/release-1.22/release-notes/maps/pr-102108-map.yaml

```diff
@@ -0,0 +1,5 @@
+pr: 102108
+releasenote:
+  text: 'kubead: remove the deprecated `--csr-only` and `--csr-dir` flags from `kubeadm
+    #     init phase certs`. Deprecate the same flags under `kubeadm certs renew`.
```

Comment:
Spacing is a bit off also you must remove the `#` from this and below line.

Reply:
Done